cmd/meroxa/root/apps/deploy.go

```diff
@@ -600,7 +606,45 @@ func (d *Deploy) prepareAppName(ctx context.Context) string {
 	return appName
 }
 
+func (d *Deploy) waitForDeployment(ctx context.Context) error {
+	startTime := time.Now().UTC()
+	logs := []string{}
+	for time.Now().UTC().Sub(startTime) < 5*time.Minute {
```

Comment:
Is this running every ms or how often?

Reply:
can we use a ticket with a context timeout instead?
kind of looks like this
```
	cctx, cancel := context.WithTimeout(ctx, <timeout>)
	defer cancel()

	t := time.NewTicker(1 * time.Second)
	defer t.Stop()

	for {
		select {
		case <-t.C:
			<do work>
		case <-cctx.Done():
                       <timeout expired>
			return cctx.Err()
		}
	}
```

Reply:
yeah, it doesn't sleep or backoff at all, Raul, a small sleep might be good.  Given that deployments are immediately done with no output logs, this doesn't do much of anything now.  mostly exercised by the unit test.